### ISSUE TITLE
Can't get audio out of lv2. linux mint 20 cinnamon 64 bit, carla and reaper.

### ISSUE THREAD

**Issue by TheGuyWhoo** [timothyschoen/PlugData#124]
At first I thought it might be due to midi not getting into the plugin, but I tested that and notein is definitely working correctly.

Here is the specific patch, I can get it to work in vst versions but not lv2.

[sampler.pd.zip](https://github.com/timothyschoen/PlugData/files/8749091/sampler.pd.zip)

(param1 = attack, param2 = decay, param3 = sustain, param 4 = release)

Also as a side note, one time with the lv2 I got some loud audio from it when I loaded it in reaper. I always have a limiter on my master, and thought it might a good idea to suggest others to protect themselves with a limiter. I've been unable to reproduce it, so not sure what caused it.

**Comment by timothyschoen** [comment]
I'll look into this when I'm done with rewriting some parts!

**Comment by timothyschoen** [comment]
Also, juce7 with built-in LV2 support is coming soon, so I'm guessing that will fix it.

**Comment by timothyschoen** [comment]
I got some unexplicable compile errors for the LV2 builds, so I upgraded to the juce7 preview as I was losing hope. It might cause some minor bugs (doubt it though!), but it should also resolve most issues with the LV2 plugin. I'm curious to hear if it makes a difference!

Btw, I'd always recommend using a limiter in a DAW, and using small speakers for the standalone. This is a recommendation for all patching software I guess, sometimes signals can do something you don't expect!
